pkg/emailer/service.go

```diff
@@ -62,3 +75,28 @@ func (s *EmailerService) SendPendingEmail(inst *instance.Instance, cmd *SendEmai
 
 	return err
 }
+
+func (s *EmailerService) SendMarketingEmail(inst *instance.Instance, cmd *SendEmailCmd) error {
+	if cmd.Subject == "" {
+		return fmt.Errorf("Missing email subject")
+	}
+	if cmd.Parts == nil {
+		return fmt.Errorf("Missing email content")
+	}
+
+	msg, err := job.NewMessage(map[string]interface{}{
+		"mode":    mail.ModeMarketing,
+		"subject": cmd.Subject,
+		"parts":   cmd.Parts,
```

Comment:
Are we using the right layout? The stack has 2 layouts for emails, but I don't remember which one should be used in what cases

Reply:
The layout is not specified when creating these jobs in Ecolyo so I believe we use the default one (and so do we here).